changelog/7899.feature.rst

```diff
@@ -0,0 +1 @@
+Added :meth:`sunpy.net.fido_factory.UnifiedResponse.show_in_notebook` to render fido search results and :meth:`sunpy.net.attr.AttrMeta.search` to render attrs tables in form of interactive table using ``itables`` in Jupyter notebooks.
```

Comment:
Why the difference in names?

Reply:
`a.Instrument.search()` just makes more sense as the only reason we are rendering it using `itables` is because we are trying to search but `show_in_notebook` can be mostly used to make the render pretty and I also believe these names makes them easy to remember.

Reply:
Does search work in a python terminal? It sounds like it should.

Reply:
Oh, okay so shall I rename it to 'show_in_notebook' as well?

Reply:
Yes please.

Reply:
Yes added, can you help me with the failing ref :meth:`sunpy.net.attr.AttrMeta.show_in_notebook`, is it wrong?

Reply:
The Metaclass is not documented in the API: https://docs.sunpy.org/en/stable/reference/net.html#id38

Reply:
Thank you, added.